requirements.pip

```diff
@@ -21,3 +21,4 @@ django-sendfile==0.3.11
 psutil>=5.4.0
 beautifulsoup4==4.6.0
 requests==2.9.1
+deprecated==1.1.0
```

Comment:
Do we benefit from importing an entire module rather than just adding the single line of code which that's actually running:

`warnings.warn('…legacy code path hit…', category=DeprecationWarning)`

Reply:
One factor to consider here is whether we want a one-time-only log message or to either log each request – with e.g. the referring URL for diagnostic purposes – or simply use varnishlog to look for this?